inc/field.class.php

```diff
@@ -1033,9 +1033,9 @@ function () {
                         'change',
                         'input, select, textarea',
                         function(evt) {
-                            if (evt.target.name == "itilcategories_id") {
+                            if (evt.target.name == "itilcategories_id" && {$items_id} == 0) {
```

Comment:
```suggestion
                            if (evt.target.name == "itilcategories_id" && $items_id == 0) {
```

Reply:
Are you sure ? other php variable are enclosed by `{}`

Reply:
I also think it should be enclosed, because it's PHP that generates JS.

Reply:
Yes, indeed, my bad

Reply:
:beers: